imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.java

```diff
@@ -34,6 +38,8 @@ public ImagePickerConfig(Context context) {
         this.folderMode = false;
         this.imageDirectory = context.getString(R.string.image_directory);
         this.returnAfterCapture = true;
+        this.actionBarColor = Color.parseColor("#3F51B5");
```

Comment:
I think `actionbarColor` and `statusBarColor` should not have default color, since it will take Application **Primary** and **Accent** color. 

If any, it should be something like `-1` or in more declarative way 
```java
public static final int UNDEFINED = -1;
```